pkg/operator/nfs/controller.go

```diff
@@ -154,17 +153,40 @@ func createServicePorts() []v1.ServicePort {
 	}
 }
 
+func (c *Controller) createStorageClass(nfsServer *nfsServer) error {
```

Comment:
Is this the storage class that will be referenced by the PVC to consume NFS storage?

Reply:
yes